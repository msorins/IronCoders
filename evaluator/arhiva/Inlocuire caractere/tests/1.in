5 2
a a a a
b b b b
z z z z
ab ab ab
ba ba ba
a b
b c
mere pere
