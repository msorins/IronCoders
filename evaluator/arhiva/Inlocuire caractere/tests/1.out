c c c c
c c c c
z z z z
ab ab ab
ba ba ba